src/binaryen-c.h

```diff
@@ -2124,13 +2124,13 @@ BINARYEN_API BinaryenIndex BinaryenGetOneCallerInlineMaxSize(void);
 // Applies to all modules, globally.
 BINARYEN_API void BinaryenSetOneCallerInlineMaxSize(BinaryenIndex size);
 
-// Gets whether heavyweight functions are allowed to be inlined.
+// Gets whether functions with loops are allowed to be inlined.
 // Applies to all modules, globally.
-BINARYEN_API int BinaryenGetAllowHeavyweight(void);
+BINARYEN_API int BinaryenGetAllowFunctionsWithLoops(void);
 
```

Comment:
Sorry for not noticing this sooner, but these should have Inlining in the name, like on line 2125. Otherwise the name isn't clear enough. Maybe `BinaryenGetAllowInliningFunctionsWithLoops`.

Reply:
Oh, right. Thanks to point me on this! Done!